codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/QuicClientSessionCache.java

```diff
@@ -214,7 +214,7 @@ private static final class HostPort {
         private final String host;
         private final int port;
 
-        HostPort(String host, int port) {
+        HostPort(@Nullable String host, int port) {
```

Comment:
I think this is not correct: 
```suggestion
        HostPort(String host, int port) {
```

Reply:
If so, shouldn't the condition below be `||` instead of `&&`
https://github.com/netty/netty-incubator-codec-quic/blob/d8977034fae8224d035986e9343c87d27f5ab8b0/codec-classes-quic/src/main/java/io/netty/incubator/codec/quic/QuicClientSessionCache.java#L172-L177

Reply:
Oh wait... never mind its correct